sounds/templatetags/display_sound.py

```diff
@@ -114,10 +114,10 @@ def sound_object_retrieved_using_bulk_query_id(sound):
             'request': request,
             'is_explicit': sound_obj.is_explicit and
                            (not request.user.is_authenticated or not request.user.profile.is_adult),
-            'is_authenticated': request.user.is_authenticated(),
+            'is_authenticated': request.user.is_authenticated == True,
```

Comment:
If this is a `bool` we do not need to compare to `True`

Reply:
This is quite interesting. Some cache-related tests were failing because of that, and what I saw is that is_authenticated was not really a `bool` but a cached bool, and that would make tests fail because `is_authenticated` is used to construct cache keys. So I need the comparison to make this a normal `bool`.

However, now I don't see these errors anymore, even if removing the comparison. It is quite strange because I swear I those tests were failing and it actually took quite a long time for me to track down the issue :S